doc/integrator/print.rst

```diff
@@ -19,8 +19,38 @@ list of options the plugin can receive.
 
 *The main viewer includes a Print plugin by default.*
 
-MapFish Print configuration
----------------------------
+Start with MapFish Print v3
```

Comment:
Is this title relevent? Why "start"? What about something like "Using MapFish Print v3"?